### PR TITLE
feat: add padding less variable for Alert

### LINKED ISSUE #38346: Make Alert padding consumed from variables
### What problem does this feature solve?

Customizing the Alert component.
Like many other components the padding is determined by other variables in the system and by explicit variable for the component.

### What does the proposed API look like?

Current:
  padding: 8px 15px;

Wanted:
 padding: @alert-padding-vertical @alert-padding-horizontal;

### PR DESCRIPTION
**Pull request by madocto**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
close: #38346

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    add padding less variable for Alert     |
| 🇨🇳 Chinese |    Alert 增加 padding less 变量       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/style/themes/default.less [hunk 1]
```diff
@@ -966,6 +966,8 @@
 @alert-text-color: @text-color;
 @alert-close-color: @text-color-secondary;
 @alert-close-hover-color: @icon-color-hover;
+@alert-padding-vertical: @padding-xs;
+@alert-padding-horizontal: 15px;
```
[the remark this answers is not in the document]
Comment (afc163) [reply components/style/themes/default.less:970]: ping

[the remark this answers is not in the document]
Comment (imoctopus) [reply components/style/themes/default.less:970]: 看是15px，是不是应该`@padding-md - 1px`？

[the remark this answers is not in the document]
Comment (afc163) [reply components/style/themes/default.less:970]: 你是对的

[the remark this answers is not in the document]
Comment (imoctopus) [reply components/style/themes/default.less]: done

**Review by afc163 — APPROVED** [review]

**Pull request merged**